src/provider/camunda-platform/properties/ConditionProps.js

```diff
@@ -333,6 +339,93 @@ function Resource(props) {
   />;
 }
 
+function VariableEventProps(props) {
+  const entries = [];
+
+  entries.push({
+    id: 'conditionVariableName',
+    component: VariableName,
+    isEdited: isTextFieldEntryEdited
+  });
+
+  entries.push({
+    id: 'conditionVariableEvents',
+    component: VariableEvents,
+    isEdited: isTextFieldEntryEdited
+  });
+
+  return entries;
+}
+
+
+function VariableName(props) {
+  const {
+    element
+  } = props;
+
+  const commandStack = useService('commandStack');
+  const translate = useService('translate');
+  const debounce = useService('debounceInput');
+
+  const getValue = () => {
+    return getConditionalEventDefinition(element).get('variableName');
+  };
+
+  const setValue = (value) => {
+    commandStack.execute('element.updateModdleProperties', {
+      element: element,
+      moddleElement: getConditionalEventDefinition(element),
+      properties: {
+        variableName: value || ''
+      }
+    });
+  };
+
+  return <TextFieldEntry
+    element={ element }
+    id="conditionVariableName"
+    label={ translate('Variable Name') }
```

Comment:
We want with sentence case with our new propPanel

```suggestion
    label={ translate('Variable name') }
```